app/components/draggable-object-target.js

```diff
@@ -3,8 +3,10 @@ import Droppable from 'ember-drag-drop/mixins/droppable';
 import log from '../helpers/log';
 
 export default Ember.Component.extend(Droppable, {
-  classNames: ["draggable-object-target"],
-  
+  classNameBindings: ['droppableClass'],
+
+  droppableClass: 'draggable-object-target',
```

Comment:
@patrickberkeley in theory you could do something like this
```htmlbars
{{#draggable-object-target droppableClass=droppableMergeableClass action="mergeDatasetArray"}}
     <span>Merge</span>
{{/draggable-object-target}}
```

Where <code>droppableClass</code> is a property on the surrounding context. If you want to use this functionality including the actions, you can use <code>"ember-drag-drop": "git://github.com/sirzach/ember-drag-drop.git#feature/small-improvements",</code>